.github/workflows/post-process.yml

```diff
@@ -48,7 +52,7 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v5
         with:
-          python-version: ${{ matrix.python-version }}${{ matrix.dev }}
+          python-version: ${{ matrix.version.python }}${{ matrix.dev }}
```

Comment:
Thanks @abhardwaj73 ! Studying the `pymatgen` PR more closely, I was about to say that you need to pass the `resolution` argument into the matrix call here, e.g.

```
name: Install dependencies
        run: |
          pip install uv
          uv pip install '.[${{ matrix.version.extras }}]' --system --resolution=${{ matrix.version.resolution }}
```

But then I realized that `pymatgen` is using `uv`, not `pip` to install dependencies, and that package is what's receiving the argument (not `pip` - `pip` actually doesn't have a `resolution` option). 

I was not familiar with uv, but it appears to be a [much faster alternative to pip](https://github.com/astral-sh/uv), much like how `ruff` is a much faster linter than `flake8` or `pycodestyle`. It could be beneficial to use in unit tests because it will make them run faster (by installing deps faster). Plus obviously make it possible to use these different resolution strategies.

Can you update this to install `uv`, similar to the `pymatgen` workflow, and then pass it the resolution argument?